ui/src/main/javascript/app/core/directives/gameroom/players/players.less

```diff
@@ -0,0 +1,40 @@
+div.players-directive {
+    position: relative;
+    width: 92%;
+    height: 500px;
+    & div.positionOne {
+        position: absolute;
+        top: 40%;
+        left: 25%;
+    }
+    & div.positionTwo {
+        position: absolute;
```

Comment:
Remove `position: absolute`, you be using `& profile-thumbnail { position: absolute; }` for this tag.